### DISCUSSION TITLE
translate function does not handle markup in text

### DISCUSSION THREAD

**Discussion by nibblesnbits** [Alexandre-Fernandez/astro-i18n#75] [Ideas]
**Describe the bug**
Translating text with markup is a bit of a pain.  It would be ideal if there were a capability similar to [this](https://next-intl-docs.vercel.app/docs/usage/messages#rich-text) to allow me to include markup in my text.

**To Reproduce**
1. Clone https://github.com/nibblesnbits/astro-i18n-repro
2. `npm install`
3. `npm start`
4. See title at top of page

**Expected behavior**
The current behavior *is* expected, but there should be a way to have it optionally process the markup as markup rather than text.

**Mandatory reproduction repository**
https://github.com/nibblesnbits/astro-i18n-repro

**Comment by Alexandre-Fernandez** [comment]
@nibblesnbits Hi, I don't understand the problem or feature you are asking for, you can already have html inside your translations.

**Reply by nibblesnbits** [reply]
Check out the repro repo.  I would expect that the title contain a line break rather than the tag for the line break.

**Reply by Alexandre-Fernandez** [reply]
you need to use [`set:html`](https://docs.astro.build/en/reference/directives-reference/#sethtml)
